### DISCUSSION TITLE
Fall 2021 Release

### DISCUSSION THREAD

**Discussion by sgoggins** [chaoss/wg-evolution#430] [General]
## What's Changed
* Repo cleanup by @vchrombie in https://github.com/chaoss/wg-evolution/pull/396
* Standardized Repo Structure by @yash2002109 in https://github.com/chaoss/wg-evolution/pull/406
* Added full code of conduct by @seun-beta in https://github.com/chaoss/wg-evolution/pull/409
* Delete inessential metric folders & Update README.md by @yash2002109 in https://github.com/chaoss/wg-evolution/pull/407
* #410 Create draft Contribution Attribution metric and visualizations by @mtift in https://github.com/chaoss/wg-evolution/pull/411
* Fix minor issues in metrics for proper display of PDF (MARS) by @yash2002109 in https://github.com/chaoss/wg-evolution/pull/413
* Standardize & remove extra content from focus-area READMEs  by @yash2002109 in https://github.com/chaoss/wg-evolution/pull/414
* Correct meeting time. Fixes https://github.com/chaoss/wg-evolution/is… by @lucasgonze in https://github.com/chaoss/wg-evolution/pull/419
* Correct URL for mailing list. Fixes https://github.com/chaoss/wg-evol… by @lucasgonze in https://github.com/chaoss/wg-evolution/pull/420
* Correct mailing list URL. Fixes https://github.com/chaoss/wg-evolutio… by @lucasgonze in https://github.com/chaoss/wg-evolution/pull/421
* added contribution attribution to readme by @klumb in https://github.com/chaoss/wg-evolution/pull/417
* fixed typos, add more explanations in Issue Resolution Duration for Filters by @eyehwan in https://github.com/chaoss/wg-evolution/pull/412
* Update README.md by @vinodkahuja in https://github.com/chaoss/wg-evolution/pull/424
* Update and rename code-changes.md to code-changes-commits.md by @sgoggins in https://github.com/chaoss/wg-evolution/pull/422
* Fix name: "Issues New" by @GeorgLink in https://github.com/chaoss/wg-evolution/pull/425
* underscores to dashes to fix broken links and comply with standard. by @sgoggins in https://github.com/chaoss/wg-evolution/pull/428
* Update README.md by @sgoggins in https://github.com/chaoss/wg-evolution/pull/429

## New Contributors
* @yash2002109 made their first contribution in https://github.com/chaoss/wg-evolution/pull/406
* @seun-beta made their first contribution in https://github.com/chaoss/wg-evolution/pull/409
* @mtift made their first contribution in https://github.com/chaoss/wg-evolution/pull/411
* @lucasgonze made their first contribution in https://github.com/chaoss/wg-evolution/pull/419
* @eyehwan made their first contribution in https://github.com/chaoss/wg-evolution/pull/412
* @vinodkahuja made their first contribution in https://github.com/chaoss/wg-evolution/pull/424

**Full Changelog**: https://github.com/chaoss/wg-evolution/compare/0.5.0...release-2021-10

<hr /><em>This discussion was created from the release <a href='https://github.com/chaoss/wg-evolution/releases/tag/release-2021-10'>Fall 2021 Release</a>.</em>

**Comment by GeorgLink** [comment]
Thanks @sgoggins . How about creating a "Release" on this GitHub repo?
